src/clj/rems/api/catalogue.clj

```diff
@@ -2,19 +2,25 @@
   (:require [compojure.api.sweet :refer :all]
             [rems.api.schema :refer :all]
             [rems.api.util] ; required for route :roles
+            [rems.db.catalogue :as catalogue]
```

Comment:
this line is probably unnecessary, the rems.api.services.catalogue line below is correct